src/sys/socket/mod.rs

```diff
@@ -2009,12 +2009,116 @@ pub fn socketpair<T: Into<Option<SockProtocol>>>(
     unsafe { Ok((OwnedFd::from_raw_fd(fds[0]), OwnedFd::from_raw_fd(fds[1]))) }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct Backlog(i32);
+
+impl Backlog {
+    /// Sets the listen queue size to system `SOMAXCONN` value
+    #[cfg(target_os = "redox")]
+    pub const DEFAULT: Self = Self(128);
+    #[cfg(not(target_os = "redox"))]
+    pub const DEFAULT: Self = Self(libc::SOMAXCONN);
```

Comment:
Why are we calling this value `DEFAULT`? But not `MAX` or simply `SOMAXCONN`

Reply:
It is mostly what people use we can call it differently sure.